python/versions.bzl

```diff
@@ -118,21 +122,25 @@ def get_release_url(platform, python_version, base_url = DEFAULT_RELEASE_BASE_UR
         tool_versions: A dict listing the interpreter versions, their SHAs and URL
 
     Returns:
-        filename and url pair
+        A tuple of (filename, url, and archive strip prefix)
     """
 
     url = tool_versions[python_version]["url"]
 
     if type(url) == type({}):
         url = url[platform]
 
+    strip_prefix = tool_versions[python_version].get("strip_prefix", None)
+    if type(strip_prefix) == type({}):
```

Comment:
nit: https://docs.aspect.build/bazelbuild/bazel-skylib/1.1.1/docs/types_doc_gen.html#types.is_dict is slightly better

Reply:
That would require introducing a dependency on `bazel-skylib` or vendoring this file which I think would be better done in an explicit cleanup PR. Does that sound reasonable?

Reply:
sure - but I'm pretty sure rules_python already requires users to have bazel-skylib installed? agree we shouldn't add a new dependency

Reply:
I'm not aware of that being a requirement. If that's the case, it'd be nice if there was a `python_rules_dependencies` macro that loaded all dependencies (adhering to [bazel-contrib](https://github.com/bazel-contrib/rules-template/blob/f26177bcbdd9eeab373f23ca568c98704efddd3f/mylang/repositories.bzl)).